src/deploy.ts

```diff
@@ -44,6 +44,12 @@ export type DeployConfig = {
   projectId: string;
   expires: string;
   channelId: string;
+  target: string;
+};
+
+export type productionDeployConfig = {
+  projectId: string;
+  target: string;
```

Comment:
Should this be optional (`target?: string`) since it won't always be defined?

Reply:
Agree, should be optional